packages/dataviews/README.md

```diff
@@ -2,6 +2,10 @@
 
 DataViews is a component that provides an API to render datasets using different types of layouts (table, grid, list, etc.).
 
+DataViews is data agnostic, it can work with data coming from a static (JSON file) or dynamic source (HTTP Request) — it just requires the data to be an array of objects that have an unique identifier. Consumers are responsible to query the data source appropiately based on the DataViews props:
+
+![DataViews flow](dataviews-flow.png "DataViews flow")
```

Comment:
I'm not sure this is the right way to embed images in the handbook. Do they still need to be in a external source (for example, pointing to the image as stored in this GitHub PR)?

Reply:
Also, I'm more than happy to collaborate and improve that graph: it's something I just created with the tools I had but other people may be able to create a better one.

Reply:
For the existing image we have in the handbook, we use the full URL `https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/explanations/architecture/assets/modules.png`. Not sure why or if it's better though.

Also, we do keep the "source" `modules.excalidraw` file in the repo to be able to edit it in the future if needed.

Reply:
> Also, we do keep the "source" modules.excalidraw file in the repo to be able to edit it in the future if needed.

I can explore redoing the graph using excalidraw. Happy if anyone else beats me to it (have a bit of an hectic week).

Reply:
No need to redo it if you lost your source or did it with some other tool, it was just a suggestion based on what I did before :)

Reply:
I just used [Obsidian Canvas](https://obsidian.md/canvas) and I have the source file that I can share. I'm just not sure how useful it'd be to anyone else? 😅 It seems excalidraw would be a more widely known format/app, and I agree having a source file to re-generate the image from is very useful.

Reply:
The best way to use images in the Handbook is to upload them to the Media Library for the Handbook's WordPress installation and reference it from that URL (see [example](https://github.com/WordPress/gutenberg/blob/dac8767321030e3a69111882178060bf3f2ed4c3/docs/getting-started/fundamentals/file-structure-of-a-block.md?plain=1#L7))

Happy to help in uploading the required images to the Handbook's WP installation.

There are several excalidraw diagrams in the Block Editor docs. The original excalidraw link (where things can be modified) has usually been added to a Resources list at the end of the page

_Note: For accesibility issues the link to the image containing the diagram should also point to an image_

Reply:
I added the image to the https://developer.wordpress.org/files so it can be referenced from there
https://developer.wordpress.org/files/2024/09/368600071-20aa078f-7c3d-406d-8dd0-8b764addd22a.png

Reply:
ok, pushed those changes (be welcome to push them yourself in the future!). In the interest of shipping small improvements regularly, I'm merging this one. Perhaps we need to revisit it to also include DataForm documentation here. At that point, I can try with excalidraw.